course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -370,25 +370,61 @@
         $scope.selectRole = function(role) {
             $scope.selectedRole = role;
         };
-        $scope.setTitle = function(id) {
+        $scope.setCourseInstanceData = function(id) {
             var ci = courseInstances.instances[id];
             if (angular.isDefined(ci)) {
-                $scope.title = ci.title;
+                $scope.courseInstance = $scope.getFormattedCourseInstance(ci)
             }
             else {
-                $scope.title = '';
                 var url = djangoUrl.reverse(
                               'icommons_rest_api_proxy',
                               ['api/course/v2/course_instances/' + id + '/']);
                 $http.get(url)
                     .success(function(data, status, headers, config, statusText) {
                         courseInstances.instances[data.course_instance_id] = data;
-                        $scope.title = data.title;
+                        $scope.courseInstance = $scope.getFormattedCourseInstance(data)
+
                     })
                     .error($scope.handleAjaxError);
             }
         };
 
+        $scope.getFormattedCourseInstance = function(ci) {
+            courseInstance = {};
+            if (ci) {
+                courseInstance['title']= ci.title;
+                courseInstance['school'] = ci.course ?
+                    ci.course.school_id.toUpperCase() : '';
+                courseInstance['term'] = ci.term ? ci.term.display_name : '';
+                courseInstance['year'] = ci.term ? ci.term.academic_year : '';
+                courseInstance['cid'] = ci.course_instance_id;
+                courseInstance['registrar_code_display'] = ci.course ?
+                ci.course.registrar_code_display +
+                ' (' + ci.course.course_id + ')'.trim() : '';
+
+                if (ci.secondary_xlist_instances &&
+                    ci.secondary_xlist_instances.length > 0) {
+                    courseInstance['xlist_status'] = 'Primary';
+                } else if (ci.primary_xlist_instances &&
+                    ci.primary_xlist_instances.length > 0) {
+                    courseInstance['xlist_status'] = 'Secondary';
+                } else {
+                    courseInstance['xlist_status'] = 'N/A';
+                }
+                var sites = ci.sites || [];
+                var siteIds =[]
+                sites.forEach(function (site) {
+                    site.site_id = site.external_id;
+                    if (site.site_id.indexOf('http') === 0) {
+                        site.site_id = site.site_id.substr(site.site_id.lastIndexOf('/')+1);
+                    }
+                    siteIds.push(site.site_id)
+                });
+                courseInstance['sites']= siteIds.length>0 ? siteIds.join(', ') : 'N/A';
```

Comment:
Didn't make them links as they weren't so in the mockups.  Just checked with Vittorio, and he doesn't want them as links in this page as (May change based on user feedback)